test/fixtures/subscriptions.yml

```diff
@@ -8,24 +8,40 @@ trialing:
 
 active:
   id: 2
-  practice_id: 1
+  practice_id: 2
   status: 'active'
   cancel_at_period_end: false
   current_period_start: <%= 2.days.ago %>
   current_period_end: <%= 28.days.from_now %>
 
-canceled:
+trialing_practice_sub:
   id: 3
-  practice_id: 1
-  status: 'canceled'
+  practice_id: 3
+  status: 'trialing'
   cancel_at_period_end: false
-  current_period_start: <%= 60.days.ago %>
-  current_period_end: <%= 30.days.ago %>
+  current_period_start: <%= Time.now %>
+  current_period_end: <%= 25.days.from_now %>
 
-past_due:
```

Comment:
Tests are failing given these fixture names changed, make sure all tests pass before committing the code?

```
Error:
SubscriptionTest#test_active_subscriptions_include_trials,_past_due,_and_active:
StandardError: No fixture named 'past_due' found for fixture set 'subscriptions'
    test/unit/subscription_test.rb:42:in `block in <class:SubscriptionTest>'

rails test test/unit/subscription_test.rb:35

E

Error:
SubscriptionTest#test_trial_ending_is_false_if_not_active:
StandardError: No fixture named 'canceled' found for fixture set 'subscriptions'
    test/unit/subscription_test.rb:15:in `block in <class:SubscriptionTest>'

rails test test/unit/subscription_test.rb:14
```